### DISCUSSION TITLE
条形图的背景有属性吗？

### DISCUSSION THREAD

**Discussion by sunWhereCode** [antvis/G2#5110] [Q&A]
<img width="355" alt="image" src="https://github.com/antvis/G2/assets/1262247/0ed95639-c202-474f-83b2-069b509ed201">

G2plot里有，但是和G2的设置的数据结构不一样

**Comment by hustcc** [comment]
原则就是使用图形去模拟背景，而不是直接设置背景。举个栗子：

![image](https://github.com/antvis/G2/assets/7856674/51ce348a-6c50-4032-b914-8bd24d243114)

```ts
// 这里是背景 shape
chart
  .interval()
  .encode('x', 'month')
  .encode('y', () => 1)
  .scale('y', { independent: true, domain: [0, 1] })
  .scale('x', { independent: true })
  .axis('y', false)
  .axis('x', false)
  .style('fill', 'red')
  .style('opacity', '0.1')
  .style('pointerEvents', 'none') // 不响应事件
  ;
```

也可以直接看官网在线 [DEMO](https://g2.antv.antgroup.com/examples/general/radial/#apple-activity)。
